projects/packages/forms/src/dashboard/components/actions-dropdown-menu/index.tsx

```diff
@@ -17,52 +17,43 @@ type ActionsDropdownMenuProps = {
 	exportData: { show: boolean };
 };
 
-const CreateFormDropdownItem = () => {
+const ActionsDropdownMenu = ( { exportData }: ActionsDropdownMenuProps ) => {
 	const { openNewForm } = useCreateForm();
+	const { showExportModal, openModal, closeModal, onExport, autoConnectGdrive, exportLabel } =
+		useExportResponses();
 
 	const analyticsEvent = useCallback( () => {
 		jetpackAnalytics.tracks.recordEvent( 'jetpack_wpa_forms_landing_page_cta_click', {
 			button: 'forms',
 		} );
 	}, [] );
 
-	const onClick = useCallback( () => {
+	const onCreateFormClick = useCallback( () => {
 		openNewForm( {
 			analyticsEvent,
 		} );
 	}, [ openNewForm, analyticsEvent ] );
 
-	return {
-		icon: plus,
-		onClick,
-		title: __( 'Create form', 'jetpack-forms' ),
-	};
-};
-
-const ExportDropdownItem = ( { onClick }: { onClick: () => void } ) => {
-	const { exportLabel } = useExportResponses();
-
-	return {
-		icon: download,
-		onClick,
-		title: exportLabel,
-	};
-};
-
-const ActionsDropdownMenu = ( { exportData }: ActionsDropdownMenuProps ) => {
-	const { showExportModal, openModal, closeModal, onExport, autoConnectGdrive } =
-		useExportResponses();
+	const controls = [
+		...( exportData.show
+			? [
+					{
+						icon: download,
+						onClick: openModal,
+						title: exportLabel,
+					},
+			  ]
+			: [] ),
+		{
+			icon: plus,
+			onClick: onCreateFormClick,
+			title: __( 'Create form', 'jetpack-forms' ),
+		},
+	];
 
 	return (
 		<>
-			<DropdownMenu
-				controls={ [
-					...( exportData.show ? [ ExportDropdownItem( { onClick: openModal } ) ] : [] ),
-					CreateFormDropdownItem(),
```

Comment:
This PR is a bit hard to read based on the diff. But in essence, the two methods here, ExportDropdownItem and CreateFormDropdownItem, were both defined in the same file above. Those have been deleted, and their logic moved into the main component so that the hooks they contained can be called in the component root.